docs/components/radio.md

```diff
@@ -171,6 +171,98 @@ const cardRadio = ref<string>('')
 </script>
 ```
 
+### cardOrientation
+
+Prop for changing card orientation. Default value is `vertical`, the alternative to which is `horizontal`.
+
+<KRadio
+  v-model="horizontalCard"
+  card
+  card-orientation="horizontal"
+  description="Choose this option if you want your APIs to only be accessible from within your private network."
+  label="Private"
+  selected-value="private"
+>
+  <KBadge appearance="success">
+    Recommended
+  </KBadge>
+</KRadio>
+<KRadio
+  v-model="horizontalCard"
+  card
+  card-orientation="horizontal"
+  description="Choose this option if you want your APIs to be publicly accessible by anyone on the internet."
+  label="Public"
+  selected-value="public"
+/>
+
+```html
+<KRadio
+  card-orientation="horizontal"
+  v-model="cardRadio"
+  description="Choose this option if you want your APIs to only be accessible from within your private network."
+  card
+  label="Private"
+  selected-value="private"
+>
+  <KBadge appearance="success">
+    Recommended
+  </KBadge>
+</KRadio>
+<KRadio
+  card-orientation="horizontal"
+  v-model="cardRadio"
+  description="Choose this option if you want your APIs to be publicly accessible by anyone on the internet."
+  card
+  label="Public"
+  selected-value="public"
+/>
+```
+
+### showCardRadio
```

Comment:
suggestion/nit: All of the other `card` props are prefixed with `card*` -- could we rename this prop to `cardRadioVisible` so that it is more similar?

Reply:
Done